extensions/wikia/FeedsAndPosts/WikiVariables.class.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Wikia\FeedsAndPosts;
+
+class WikiVariables {
+	public function get() {
+		$cacheTTL = 60; // a minute
+
+		return \WikiaDataAccess::cache(
+			wfMemcKey( 'feeds-wiki-variables' ),
```

Comment:
probably redundant, these are already memcached b/c of wikifactory

Reply:
see changes, can we still get rid of memcache?

Reply:
yep, all these variables are prefetched and cached by WFL at the start of the request

Reply:
fixed